### PR TITLE
Provider for Microsoft.Extensions.Logging

### PR DESCRIPTION
**Pull request by aholovko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -761,6 +763,92 @@ context properties in web applications.
     }
     ```
 
+Microsoft.Extensions.Logging Integration
+----------------------------------------
+
+Provider notifies the Airbrake dashboard on an exception with the help of
```
Comment (kyrylo) [inline README.md]: "on" -> "of"

File: README.md [hunk 2]
```diff
@@ -761,6 +763,92 @@ context properties in web applications.
...
+[Microsoft.Extensions.Logging](https://github.com/aspnet/Logging) methods.
+
+1. Install the `Sharpbrake.Extensions.Logging` package from NuGet (you can use "Package
+   Manager Console" from Visual Studio):
+
+   ```
+   PM> Install-Package Sharpbrake.Extensions.Logging
+   ```
+
+2. Configure the Airbrake provider through code:
```
Comment (kyrylo) [inline README.md]: I think "through code" can be omitted without harming readability.

File: README.md [hunk 3]
```diff
@@ -761,6 +763,92 @@ context properties in web applications.
...
+   (`LogLevel.Error` is the default one):
+
+   ```csharp
+   loggerFactory.AddAirbrake(Configuration.GetSection("Airbrake"),
+       app.ApplicationServices.GetService<IHttpContextAccessor>(),
+       LogLevel.Warning);
+   ```
+
+   More advanced setup can be used to better suit your needs. Here is an example of
+   how to define the notifier with additional filtering rules:
```
Comment (kyrylo) [inline README.md]: "the" -> "a"

File: README.md [hunk 4]
```diff
@@ -761,6 +763,92 @@ context properties in web applications.
...
+       "ProjectId": "113743",
+       "ProjectKey": "81bbff95d52f8856c770bb39e827f3f6"
+     }
+   }
+   ```
+
+   To get access to the HTTP content `app.ApplicationServices.GetService<IHttpContextAccessor>()`
+   is used.
+
+   You can set the minimum log level for exceptions being sent to the dashboard
```
Comment (kyrylo) [inline README.md]: I would write:

"Minimum log level can be set here to filter exceptions with lower severities (defaults to `LogLevel.Error`).

File: README.md [hunk 5]
```diff
@@ -761,6 +763,92 @@ context properties in web applications.
...
+   (`LogLevel.Error` is the default one):
```
Comment (kyrylo) [inline README.md]: "(defaults to `LogLevel.Error`)"

File: src/Sharpbrake.Extensions.Logging/AirbrakeLogger.cs [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+                return Severity.Warning;
+
+            if (logLevel == LogLevel.Information)
+                return Severity.Info;
+
+            if (logLevel == LogLevel.Debug)
+                return Severity.Debug;
+
+            if (logLevel == LogLevel.Trace)
+                return Severity.Notice;
```
Comment (kyrylo) [inline src/Sharpbrake.Extensions.Logging/AirbrakeLogger.cs]: Would it make sense to specify a table of log levels from Logging that correspond to Airbrake log levels? We support more than 6 log levels, so users with that knowledge might not know that Trace defaults to Notice.

Comment (aholovko) [reply src/Sharpbrake.Extensions.Logging/AirbrakeLogger.cs]: It makes sense. Perhaps it should be done for other integrations as well. Could we postpone that for another PR? I would like to perform some additional changes to documentation as well.

**Comment by aholovko** [comment]
@kyrylo PTAL

File: src/Sharpbrake.Extensions.Logging/AirbrakeLogger.cs (hunk 6 above)
Comment (kyrylo) [reply src/Sharpbrake.Extensions.Logging/AirbrakeLogger.cs]: Sounds good to me.

**Pull request merged**